### PR TITLE
BI-3868 Create skeleton base application

### PR DESCRIPTION
**Pull request by markpit**
A running base app as a starting point

had to put some TODO's in www.ts to add logging when we introduce a logger.

npm run build
npm run devstart 3000

3000 is any port to specify for running locally
'make' should work as well

localhost:3000/strike-off-objections/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "@types/express": "^4.16.1",
+    "@types/nunjucks": "^3.1.1",
+    "node-sass": "^4.14.1",
+    "nodemon": "^2.0.3",
+    "sonarqube-scanner": "^2.6.0",
+    "ts-node": "^8.0.2",
+    "tslint": "^5.20.1",
+    "typescript": "^3.3.1"
+  },
+  "license": "ISC",
```
Comment (ICoxCH) [inline package.json:32]: Is this the correct license? Repo is made with MIT license. 
Other than this all happy

File: src/routes/routes.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { Router } from "express";
+
+const router: Router = Router();
+
+router.get("/", (req, response) => response.send("This is the Strike Off Objections app &#128516;"));
```
Comment (mwestacott) [inline src/routes/routes.ts:5]: What's the magic incantation at the end of the line? Perhaps describe in a comment.

Comment (markpit) [reply src/routes/routes.ts:5]: you'll have to run it and see

Comment (markpit) [reply src/routes/routes.ts]: it's gone now :(

**Pull request merged**